components/lock.rst

```diff
@@ -227,6 +227,10 @@ PHP process is terminated::
     // the argument is the path of the directory where the locks are created
     $store = new FlockStore(sys_get_temp_dir());
```

Comment:
Given the new note below maybe we should use a custom path for the example (i.e `'/var/stores'`).